gatsby-node.js

```diff
@@ -1,8 +1,7 @@
 const path = require(`path`);
 const { execSync } = require('child_process');
 const { createFilePath } = require('gatsby-source-filesystem');
-
-const MAX_RESULTS = 5;
+const observabilityPacks = require('./src/data/observability-packs.json');
```

Comment:
These should already be in Gatsby's graphql in `allObservabilityPacks` type

Reply:
good point! i started before that was hooked up